build-tools/automation/azure-pipelines.yaml

```diff
@@ -322,7 +359,7 @@ stages:
   # Check - "Xamarin.Android (Test API Compatibility)"
   - job: mac_api_compat
     displayName: API Compatibility
-    pool: $(HostedMacMojave)
+    pool: $(HostedMac)
```

Comment:
Could we restore the `HostedMacMojave` variable? We'll still want to use it for certain test stages, rather than reverting to the generic High Sierra pool.

Reply:
Resolved